GameEngine/Players/GreedyPlayer.cs

```diff
@@ -1,13 +1,15 @@
-﻿namespace GameEngine.Players
+﻿using System.Collections.Generic;
+
+namespace GameEngine.Players
 {
-    public class GreatestDistanceSingleCardPlayer : Player
+    public class GreedyPlayer : Player
     {
         public override Play FormulatePlay(GameBoard board)
         {
             Play greatestSingleDistancePlay = null;
             int greatestDistance = 0;
             foreach (var owlPosition in board.Owls.ListOfPositions) {
-                foreach (var card in Hand)
+                foreach (var card in Hand.Cards)
```

Comment:
💡 What are your thoughts on having `PlayerHand` implement `IEnumerable` so you can iterate over the Hand, not Hand.Cards?